tools/app/app/cards/[key]/edit/page.tsx

```diff
@@ -36,7 +38,9 @@ export default function Page({ params }: { params: { key: string } }) {
   const { project } = useProject()
   const { card, updateCard } = useCard(params.key)
   const { fieldTypes } = useFieldTypes()
-  const { cardType } = useCardType(card?.metadata?.cardtype ?? null)
+  const cardType = useMemo(() => {
+    return project?.cardTypes.find((ct) => ct.name === card?.metadata?.cardtype)
```

Comment:
There's also `project.cardType(<name>)` if you want all of the details of the card-type.

Reply:
The project-object of the frontend contains all details of cardtypes. See https://github.com/CyberismoCom/cyberismo/blob/d78993a479c52afe22737695921450f9bf80b20f/tools/app/app/api/cards/route.tsx#L21